app/controllers/pages_controller.rb

```diff
@@ -11,6 +11,11 @@ def dashboard
     @freediving_articles = Article.joins(:folders).where(folders:{title:"Freediving"}, status:["missed", "to_read"]).order("reminder ASC NULLS LAST")
     @neuroscience_articles = Article.joins(:folders).where(folders:{title:"Neuroscience"}, status:["missed", "to_read"]).order("reminder ASC NULLS LAST")
     @tennis_articles = Article.joins(:folders).where(folders:{title:"Tennis"}, status:["missed", "to_read"]).order("reminder ASC NULLS LAST")
+    @articles = Article.all
+    @missed_article = Article.missed.order("reminder DESC NULLS LAST")
+    #ordering descendant
+    #@next_article = Article.where("#{reminder.to_date.to_s} > #{Date.today.to_s}")
```

Comment:
Change it to:
```ruby
@next_articles = Article.where("reminder > ?", Time.zone.now)
```
And it should be what you want

Reply:
@lisamariemaass , [you can also make it into a scope](https://github.com/andrerferrer/active-record-scope-demo#goal)

@Brunzly @VincenzoGalante , marking you **FYI**

Reply:
Source -> https://api.rubyonrails.org/classes/ActiveSupport/TimeWithZone.html